### DISCUSSION TITLE
Discuss: Cleared for Takeoff – Embracing Open Source within the Enterprise

### DISCUSSION THREAD

**Discussion by jeffreality** [community/community#142848] [Other Feature Feedback, Questions, & Ideas]
# Cleared for Takeoff: Embracing Open Source within the Enterprise
### Universe Session Track: DevOps — DEV1460B
*Tuesday, October 29th, 12:30PM-01:10PM*

In today’s rapidly evolving tech landscape, open source isn’t just a buzzword — it’s a catalyst for collaboration, agility, and transparent innovation. This session dives into Southwest Airlines' journey in embracing open source to drive innovation and build community. We’ll discuss how open-source practices break down silos, foster collaboration, and create solutions that are both agile and secure. Attendees will learn strategies for securing **executive buy-in**, managing **legal** considerations, and fostering an active open-source **community** within an enterprise.

Facilitator: Jeffrey Berthiaume, *Sr. Innovator*, **Southwest Airlines**

Let’s Get This Discussion Off the Ground!
I’d love to hear your thoughts: *What challenges or successes have you experienced in persuading your company that releasing internally developed code as open source is secure and beneficial?*

Jump in to share your experiences with building internal or external communities, managing open-source contributions, or securing buy-in from stakeholders. Let’s keep the ideas grounded in real-world examples, exploring both the potential and practicalities of open-source in the enterprise.

And if you’re at Universe, please join the session to explore some templates I’ll be sharing, and bring your own experiences to the Q&A. I’ll also be available to chat afterward. Looking forward to connecting!

**Comment by tmtri3102** [comment]
So excited! Can't wait

**Comment by jeffreality** [comment]
Here are the two templates I wanted to share with everyone who caught my presentation—thank you for tuning in! I'd love to hear your thoughts: Did these help? What did you add or adjust? And did they help spark your own internal open source initiatives?

[OSS-Triage.docx](https://github.com/user-attachments/files/17608899/OSS-Triage.docx)
[OSS-Dependencies.xlsx](https://github.com/user-attachments/files/17608900/OSS-Dependencies.xlsx)

**Comment by jeffreality** [comment]
Also, if you didn't get a chance to see it in person, here is a link to my talk: https://www.youtube.com/watch?v=HLXG-57o7NE

**Comment by 07SD** [comment]
<img width="1080" height="1080" alt="AutoTune_Hip-Hop_Artwork" src="https://github.com/user-attachments/assets/667cf1f7-a21e-4eb1-8edf-d4e22ce9c1b4" />
